docker/build_images.sh

```diff
@@ -196,4 +199,8 @@ if [ -n "${REBUILD_DATASETS}" -a "${REBUILD_DATASETS}" != "n" ]; then
 
     # remove the versioned file
     rm Dockerfile.datasets_version	
+fi
+
+if [ "${UPDATE_PUBLIC_DATA}" == "Y" ]; then
+    python update_grein.py
```

Comment:
I believe that the command misses the required parameters. Don't forget that this is being run within the build environment and not the docker container. Therefore, the environment variables (and the respective paths) are not available.

You could use a relative path for simplicity such as `--path_whitellist ../reactome_analysis_api/reactome_analysis_api/searcher/whitelist.txt`